app/townsquare/templates/townsquare/index.html

```diff
@@ -148,7 +148,7 @@ <h5 class="font-weight-semibold font-subheader mb-1">{{nb.title}}</h5>
                     <div class="checkbox_container">
                       <input name="trending" id="trending" type="checkbox" value="0" val-ui='Trending' {% if trending_only %}checked=checked{%endif%} />
                       <span class="checkbox"></span>
-                      <div class="filter-label ml-1">
+                      <div class="filter-label ml-1 mr-2">
```

Comment:
also for the sake of uniformity, let's remove the `mr-2` from here

Reply:
oh nice catch ! Did not see that